index.html

```diff
@@ -3187,7 +3187,15 @@ <h3 id="h-sourcebufferlist-events" resource="#h-sourcebufferlist-events"><span p
         </thead>
         <tbody>
           <tr>
-            <td>06 June 2016</td>
+            <td>07 June 2016</td>
```

Comment:
rebase to current gh-pages and regenerate please, so the other recent changes aren't lost.